src/Components/SeparationBox/SeparationBox.tsx

```diff
@@ -0,0 +1,25 @@
+import { Box, Stack, Typography, styled } from '@mui/material'
+
+const SeparationBox = () => {
+  const DivBox = styled(Box)(({ theme }) => ({
+    height: '0.0625rem',
+    width: '4rem',
+    backgroundColor: theme.palette.blackGrey.main
+  }))
+
+  const SpanTypography = styled(Typography)(({ theme }) => ({
+    color: theme.palette.blackGrey.main,
+    marginRight: '0.25rem',
+    marginLeft: '0.25rem'
+  }))
```

Comment:
let's put styles in styles.ts, for each component you have directory in this directory let's create for each component styles.ts